hathor/merged_mining/coordinator.py

```diff
@@ -624,7 +624,10 @@ def handle_submit(self, params: list[Any], msgid: Optional[str]) -> None:
 
         try:
             aux_pow = job.build_aux_pow(work)
-            aux_pow.verify(block_base_hash)
+            # This constant usually comes from full node settings, but it's hardcoded here as this runs as a
+            # standalone CLI tool.
+            old_max_merkle_path_length = 12
```

Comment:
Add a note that it must be equal to `settings.NEW_MAX_MERKLE_PATH_LENGTH`.

Reply:
I'll separate the aux_pow verify method and not verify the merkle path length here

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/910/commits/8aa74cba4e8edb7108bf64d45c7af7ba7adca292

Reply:
Great! Thanks!